enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/CreateAcceptanceTest.scala

```diff
@@ -91,4 +91,25 @@ class CreateAcceptanceTest extends ExecutionEngineFunSuite with QueryStatisticsT
     result.executionPlanDescription()
   }
 
+  //Not TCK material
+  // This test exposed a bug in the slotted runtime where it could mix up long slots with ref slots
+  test("should not accidentally create relationship between the wrong nodes") {
```

Comment:
Testing for specific bugs are usually called _"Regression testing"_. Perhaps we should have a separate (set of) test class(es) for regression tests, because it feels a bit out of place here...

Reply:
That would be kind of nice to separate those. But we haven't had it so far in the scala acceptance tests so they are already scattered with regression tests for specific issues. Maybe we can do this as a separate test refactoring?